app/src/org/commcare/android/fragments/SetupEnterURLFragment.java

```diff
@@ -51,6 +55,21 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
         installButton = (Button) view.findViewById(R.id.start_install);
         installButton.setText(Localization.get("install.button.start"));
         prefixURLSpinner = (Spinner) view.findViewById(R.id.url_spinner);
+        prefixURLSpinner.setOnItemSelectedListener(new AdapterView.OnItemSelectedListener() {
+            @Override
+            public void onItemSelected(AdapterView<?> parent, View view, int position, long id) {
+                String selectedText = profileLocation.getText().toString();
+                if((position == prefixURLSpinner.getCount() -1) &&
+                        (selectedText == null || selectedText.length() == 0)) {
```

Comment:
Relying on position seems more brittle than, say, looking for the text "raw" (though that also isn't perfect). But the assertion below is a reasonable way to catch unintended consequences of changing the dropdown.